Formula/clusterctl.rb

```diff
@@ -1,9 +1,11 @@
 class Clusterctl < Formula
   desc "Home for the Cluster Management API work, a subproject of sig-cluster-lifecycle"
   homepage "https://cluster-api.sigs.k8s.io"
-  url "https://github.com/kubernetes-sigs/cluster-api/archive/refs/tags/v0.4.0.tar.gz"
-  sha256 "af60c8aa1b21eccc7f272c0135ffaf1a8e0ecdffb4173075efac52509ce0eeb0"
+  url "https://github.com/kubernetes-sigs/cluster-api.git",
+      tag:      "v0.4.0",
+      revision: "7f879be68d15737e335b6cb39d380d1d163e06e6"
   license "Apache-2.0"
+  head "https://github.com/kubernetes-sigs/cluster-api.git"
```

Comment:
```suggestion
  revision 1
  head "https://github.com/kubernetes-sigs/cluster-api.git"
```
We can bump the revision here if you want users to get the new version of the formula with this fix on running `brew upgrade`.

Reply:
Thank you for the feedback. I'm a new contributor and doesn't have a complete understanding of the formula and its structure. I thought since I'm simply replacing the source (without bumping the version) of the formula from the tarball to git clone URL, it's changed. Sorry about that. I will do the needful. 😄